packages/vhd-lib/Vhd/VhdFile.integ.spec.js

```diff
@@ -42,6 +42,19 @@ afterEach(async () => {
   disposeHandler()
 })
 
+test('checkFile fails with unvalid VHD file', async () => {
+  const initalSize = 4
+  const rawFileName = `${tempDir}/randomfile`
+  await createRandomFile(rawFileName, initalSize)
+  const vhdFileName = `${tempDir}/vhdFile.vhd`
+  await convertFromRawToVhd(rawFileName, vhdFileName)
+
+  await checkFile(vhdFileName)
+
+  await fs.truncate(vhdFileName, 2)
```

Comment:
`initialSize` is in MB while `truncate` accepts a size in bytes, this is confusing.